src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -118,28 +114,45 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
     [accountId, assetNamespace, assetReference, chainId, defaultAccountId, highestBalanceAccountId],
   )
 
-  const opportunityData = useAppSelector(state =>
+  const earnOpportunityData = useAppSelector(state =>
     selectEarnUserStakingOpportunityByUserStakingId(state, opportunityDataFilter),
   )
 
+  const opportunitiesMetadata = useAppSelector(state => selectStakingOpportunitiesById(state))
+
+  const opportunityMetadata = useMemo(
+    () => opportunitiesMetadata[assetId as StakingId],
+    [assetId, opportunitiesMetadata],
+  ) as OpportunityMetadata<DefiProvider.ThorchainSavers, DefiType.Staking> | undefined
+
+  const currentCapFillPercentage = bnOrZero(
+    opportunityMetadata?.opportunitySpecific?.saversSupplyIncludeAccruedFiat,
+  )
+    .div(bnOrZero(opportunityMetadata?.opportunitySpecific?.saversMaxSupplyFiat))
+    .times(100)
+    .toNumber()
+  const isCapUsed =
+    bnOrZero(opportunityMetadata?.opportunitySpecific?.saversMaxSupplyFiat).gt(0) &&
+    bnOrZero(currentCapFillPercentage).eq(100)
```

Comment:
Leftover from the view layer base work, expose and use `synth_mint_paused` instead

Reply:
446ad8bf15283d6b16646e23f92362e188ab14af